indico/modules/categories/templates/management/settings.html

```diff
@@ -27,6 +27,42 @@
 
 {% block content %}
     {{ form_header(form) }}
+        {{ form_rows(form, fields=form.TITLE_FIELDS) }}
+        {% if untranslated_title_languages != '[]' %}
+            <div id="title-translation-container">
+                <div id="target-new-title-translation-fields"></div>
+            </div>
+            <div class="form-group">
+                <div class="form-label"></div>
+                <div class="form-field">
+                    <a class="i-button icon-edit"
+                    onclick="return addTranslateTitleFields('{{untranslated_title_languages}}')"
```

Comment:
We don't really use inline JS anymore, it's usually not considered good practice nowadays ;)

But if you follow my advise with the "WTF react component" you won't need this stuff here anyway